.github/workflows/ci-release.yml

```diff
@@ -60,6 +60,15 @@ jobs:
 
           # Execute the gradlew command to publish the build
           sudo -E env "PATH=$PATH" bash -c "ulimit -l 65536 && ulimit -a && ./gradlew --no-daemon --parallel -PreleaseBuild=true$FIRST_GRADLE_TARGETS && ./gradlew --no-daemon --no-parallel -PreleaseBuild=true$SECOND_GRADLE_TARGETS"
+      - name: Central Portal manual Repo Close
+        # Details at https://central.sonatype.org/publish/publish-portal-ossrh-staging-api/#configuring-the-repository
+        env:
+          SONATYPE_USER: ${{ secrets.SONATYPE_USER }}
+          SONATYPE_TOKEN: ${{ secrets.SONATYPE_TOKEN }}
+        run: |
+          BEARER_TOKEN=printf "$SONATYPE_USER:$SONATYPE_TOKEN" | base64
+          curl -H "Authorization: Bearer $BEARER_TOKEN" -X POST https://ossrh-staging-api.central.sonatype.com/manual/upload/defaultRepository/io.servicetalk
```

Comment:
sonatype is known to be fragile from time to time, consider adding a few retries for this command in case they have intermediate errors: https://everything.curl.dev/usingcurl/downloads/retry.html

Reply:
Deal.